lib/flipper/instrumentation/bugsnag.rb

```diff
@@ -0,0 +1,21 @@
+module Flipper
+  module Instrumentation
+    class Bugsnag
+      def call(name, start, finish, id, payload)
+        operation, feature_name, result = payload.values_at(:operation, :feature_name, :result)
+        return unless operation == :enabled?
+
+        if result
```

Comment:
Neat! Any thoughts on if a feature flag returns true on a page in one spot and then false in another? For example, % of time can do this. It seems like this implementation would leave it with whatever was last. Correct?

Reply:
Correct, this implementation means last flag wins. I think that is the behavior you would want so any bugs reported show what the last value was of a feature check.